src/common-components/Zendesk.jsx

```diff
@@ -48,6 +50,10 @@ const ZendeskHelp = () => {
     },
   };
 
+  if (window.location.pathname === REGISTER_EMBEDDED_PAGE && isHostAvailableInQueryParams()) {
```

Comment:
I think we can make do without the `isHostAvailableInQueryParams`. The page embedded registration is only ever accessible if host is present.

Reply:
yes you are right